services/namex-pay/src/namex_pay/worker.py

```diff
@@ -92,7 +93,7 @@ async def update_payment_record(payment: Payment) -> Optional[Payment]:
 
     payment_action = payment.payment_action
     nr = RequestDAO.find_by_id(payment.nrId)
-    if payment_action == Payment.PaymentActions.CREATE.value:  # pylint: disable=R1705
+    if payment_action in [Payment.PaymentActions.CREATE.value, Payment.PaymentActions.RESUBMIT.value]:  # pylint: disable=R1705
```

Comment:
I know what's going because I'm reviewing this specific PR, but how about a comment here to explain to a future developer what's going on here?